main.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright (C) 2024 Nordix Foundation.
+For a full list of individual contributors, please see the commit history.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/cdevents/webhook-cdevents-adapter/pkg/cdevents"
+	"github.com/hashicorp/go-plugin"
+)
+
+func run() error {
+	translator, err := cdevents.LoadConfig("translator-plugins.yaml")
```

Comment:
I guess this is fine for a first PR, however, if we want this app to run containerised, we should support loading config from a configurable location.